<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Cucumber" time="81.153" tests="103" skipped="0" failures="46" errors="0">
<testcase classname="AdminLogin Controller" name="Check if Admin able to generate token with valid credential" time="20.081">
<system-out><![CDATA[
Given Admin creates request with valid credentials..........................passed
When Admin calls "LMSLogIn" with "Post" http request with valid endpoint....passed
Then Admin Should get 200 status............................................passed
And Adimn should get auto generated token...................................passed
]]></system-out>
</testcase>
<testcase classname="POST REQUEST for Program Module" name="Check if Admin able to create a program with &quot;&lt;scenarioName&gt;&quot; - Example #1.1" time="2.171">
<system-out><![CDATA[
Given User is Authorized....................................................passed
Given Admin creates "POST" Request for the LMS with request body "Program" and 1.passed
When Admin sends "POST" Request and request Body with endpoint for Program..passed
Then Admin receives 201 "Created" Status with response body for Program.....passed
]]></system-out>
</testcase>
<testcase classname="POST REQUEST for Program Module" name="Check if Admin able to create a program with &quot;&lt;scenarioName&gt;&quot; - Example #1.2" time="0.77">
<system-out><![CDATA[
Given User is Authorized....................................................passed
Given Admin creates "POST" Request for the LMS with request body "Program" and 1.passed
When Admin sends "POST" Request and request Body with endpoint for Program..passed
Then Admin receives 201 "Created" Status with response body for Program.....passed
]]></system-out>
</testcase>
<testcase classname="POST REQUEST for Program Module" name="Check if Admin able to create a program with &quot;&lt;scenarioName&gt;&quot; - Example #1.3" time="0.759">
<system-out><![CDATA[
Given User is Authorized....................................................passed
Given Admin creates "POST" Request for the LMS with request body "Program" and 2.passed
When Admin sends "POST" Request and request Body with endpoint for Program..passed
Then Admin receives 401 "Unauthorized" Status with response body for Program.passed
]]></system-out>
</testcase>
<testcase classname="POST REQUEST for Program Module" name="Check if Admin able to create a program with &quot;&lt;scenarioName&gt;&quot; - Example #1.4" time="0.554">
<system-out><![CDATA[
Given User is Authorized....................................................passed
Given Admin creates "POST" Request for the LMS with request body "Program" and 3.passed
When Admin sends "POST" Request and request Body with endpoint for Program..passed
Then Admin receives 404 "Not Found" Status with response body for Program...passed
]]></system-out>
</testcase>
<testcase classname="POST REQUEST for Program Module" name="Check if Admin able to create a program with &quot;&lt;scenarioName&gt;&quot; - Example #1.5" time="0.476">
<system-out><![CDATA[
Given User is Authorized....................................................passed
Given Admin creates "GET" Request for the LMS with request body "Program" and 4.passed
When Admin sends "GET" Request and request Body with endpoint for Program...passed
Then Admin receives 405 "Method Not Allowed" Status with response body for Program.passed
]]></system-out>
</testcase>
<testcase classname="POST REQUEST for Program Module" name="Check if Admin able to create a program with &quot;&lt;scenarioName&gt;&quot; - Example #1.6" time="0.574">
<system-out><![CDATA[
Given User is Authorized....................................................passed
Given Admin creates "POST" Request for the LMS with request body "Program" and 5.passed
When Admin sends "POST" Request and request Body with endpoint for Program..passed
Then Admin receives 400 "Bad Request" Status with response body for Program.passed
]]></system-out>
</testcase>
<testcase classname="POST REQUEST for Program Module" name="Check if Admin able to create a program with &quot;&lt;scenarioName&gt;&quot; - Example #1.7" time="0.54">
<system-out><![CDATA[
Given User is Authorized....................................................passed
Given Admin creates "PUT" Request for the LMS with request body "Program" and 8.passed
When Admin sends "PUT" Request and request Body with endpoint for Program...passed
Then Admin receives 200 "Ok" Status with response body for Program..........passed
]]></system-out>
</testcase>
<testcase classname="POST REQUEST for Program Module" name="Check if Admin able to create a program with &quot;&lt;scenarioName&gt;&quot; - Example #1.8" time="0.442">
<system-out><![CDATA[
Given User is Authorized....................................................passed
Given Admin creates "PUT" Request for the LMS with request body "Program" and 9.passed
When Admin sends "PUT" Request and request Body with endpoint for Program...passed
Then Admin receives 200 "Ok" Status with response body for Program..........passed
]]></system-out>
</testcase>
<testcase classname="POST REQUEST for Program Module" name="Check if Admin able to create a program with &quot;&lt;scenarioName&gt;&quot; - Example #1.9" time="0.365">
<system-out><![CDATA[
Given User is Authorized....................................................passed
Given Admin creates "PUT" Request for the LMS with request body "Program" and 10.passed
When Admin sends "PUT" Request and request Body with endpoint for Program...passed
Then Admin receives 401 "Unauthorized" Status with response body for Program.passed
]]></system-out>
</testcase>
<testcase classname="POST REQUEST for Program Module" name="Check if Admin able to create a program with &quot;&lt;scenarioName&gt;&quot; - Example #1.10" time="0.698">
<system-out><![CDATA[
Given User is Authorized....................................................passed
Given Admin creates "GET" Request for the LMS with request body "Program" and 11.passed
When Admin sends "GET" Request and request Body with endpoint for Program...passed
Then Admin receives 405 "Method Not Allowed" Status with response body for Program.passed
]]></system-out>
</testcase>
<testcase classname="POST REQUEST for Program Module" name="Check if Admin able to create a program with &quot;&lt;scenarioName&gt;&quot; - Example #1.11" time="0.531">
<system-out><![CDATA[
Given User is Authorized....................................................passed
Given Admin creates "PUT" Request for the LMS with request body "Program" and 12.passed
When Admin sends "PUT" Request and request Body with endpoint for Program...passed
Then Admin receives 404 "Not Found" Status with response body for Program...passed
]]></system-out>
</testcase>
<testcase classname="POST REQUEST for Program Module" name="Check if Admin able to create a program with &quot;&lt;scenarioName&gt;&quot; - Example #1.12" time="0.357">
<system-out><![CDATA[
Given User is Authorized....................................................passed
Given Admin creates "PUT" Request for the LMS with request body "Program" and 13.passed
When Admin sends "PUT" Request and request Body with endpoint for Program...passed
Then Admin receives 400 "Bad Request" Status with response body for Program.passed
]]></system-out>
</testcase>
<testcase classname="POST REQUEST for Program Module" name="Check if Admin able to create a program with &quot;&lt;scenarioName&gt;&quot; - Example #1.13" time="0.684">
<system-out><![CDATA[
Given User is Authorized....................................................passed
Given Admin creates "PUT" Request for the LMS with request body "Program" and 16.passed
When Admin sends "PUT" Request and request Body with endpoint for Program...passed
Then Admin receives 401 "Unauthorized" Status with response body for Program.passed
]]></system-out>
</testcase>
<testcase classname="POST REQUEST for Program Module" name="Check if Admin able to create a program with &quot;&lt;scenarioName&gt;&quot; - Example #1.14" time="0.442">
<system-out><![CDATA[
Given User is Authorized....................................................passed
Given Admin creates "GET" Request for the LMS with request body "Program" and 17.passed
When Admin sends "GET" Request and request Body with endpoint for Program...passed
Then Admin receives 405 "Method Not Allowed" Status with response body for Program.passed
]]></system-out>
</testcase>
<testcase classname="POST REQUEST for Program Module" name="Check if Admin able to create a program with &quot;&lt;scenarioName&gt;&quot; - Example #1.15" time="0.471">
<system-out><![CDATA[
Given User is Authorized....................................................passed
Given Admin creates "PUT" Request for the LMS with request body "Program" and 18.passed
When Admin sends "PUT" Request and request Body with endpoint for Program...passed
Then Admin receives 404 "Not Found" Status with response body for Program...passed
]]></system-out>
</testcase>
<testcase classname="POST REQUEST for Program Module" name="Check if Admin able to create a program with &quot;&lt;scenarioName&gt;&quot; - Example #1.16" time="0.537">
<system-out><![CDATA[
Given User is Authorized....................................................passed
Given Admin creates "PUT" Request for the LMS with request body "Program" and 19.passed
When Admin sends "PUT" Request and request Body with endpoint for Program...passed
Then Admin receives 400 "Bad Request" Status with response body for Program.passed
]]></system-out>
</testcase>
<testcase classname="POST REQUEST for Program Module" name="Check if Admin able to create a program with &quot;&lt;scenarioName&gt;&quot; - Example #1.17" time="1.255">
<system-out><![CDATA[
Given User is Authorized....................................................passed
Given Admin creates "GET" Request for the LMS with request body "Program" and 22.passed
When Admin sends "GET" Request and request Body with endpoint for Program...passed
Then Admin receives 200 "Ok" Status with response body for Program..........passed
]]></system-out>
</testcase>
<testcase classname="POST REQUEST for Program Module" name="Check if Admin able to create a program with &quot;&lt;scenarioName&gt;&quot; - Example #1.18" time="0.708">
<system-out><![CDATA[
Given User is Authorized....................................................passed
Given Admin creates "GET" Request for the LMS with request body "Program" and 23.passed
When Admin sends "GET" Request and request Body with endpoint for Program...passed
Then Admin receives 200 "Ok" Status with response body for Program..........passed
]]></system-out>
</testcase>
<testcase classname="POST REQUEST for Program Module" name="Check if Admin able to create a program with &quot;&lt;scenarioName&gt;&quot; - Example #1.19" time="0.937">
<system-out><![CDATA[
Given User is Authorized....................................................passed
Given Admin creates "GET" Request for the LMS with request body "Program" and 24.passed
When Admin sends "GET" Request and request Body with endpoint for Program...passed
Then Admin receives 200 "Ok" Status with response body for Program..........passed
]]></system-out>
</testcase>
<testcase classname="POST REQUEST for Program Module" name="Check if Admin able to create a program with &quot;&lt;scenarioName&gt;&quot; - Example #1.20" time="0.38">
<system-out><![CDATA[
Given User is Authorized....................................................passed
Given Admin creates "GET" Request for the LMS with request body "Program" and 25.passed
When Admin sends "GET" Request and request Body with endpoint for Program...passed
Then Admin receives 404 "Not Found" Status with response body for Program...passed
]]></system-out>
</testcase>
<testcase classname="POST REQUEST for Program Module" name="Check if Admin able to create a program with &quot;&lt;scenarioName&gt;&quot; - Example #1.21" time="0.374">
<system-out><![CDATA[
Given User is Authorized....................................................passed
Given Admin creates "GET" Request for the LMS with request body "Program" and 26.passed
When Admin sends "GET" Request and request Body with endpoint for Program...passed
Then Admin receives 401 "Unauthorized" Status with response body for Program.passed
]]></system-out>
</testcase>
<testcase classname="POST REQUEST for Program Module" name="Check if Admin able to create a program with &quot;&lt;scenarioName&gt;&quot; - Example #1.22" time="0.476">
<system-out><![CDATA[
Given User is Authorized....................................................passed
Given Admin creates "GET" Request for the LMS with request body "Program" and 27.passed
When Admin sends "GET" Request and request Body with endpoint for Program...passed
Then Admin receives 404 "Not Found" Status with response body for Program...passed
]]></system-out>
</testcase>
<testcase classname="POST REQUEST for Program Module" name="Check if Admin able to create a program with &quot;&lt;scenarioName&gt;&quot; - Example #1.23" time="0.355">
<system-out><![CDATA[
Given User is Authorized....................................................passed
Given Admin creates "GET" Request for the LMS with request body "Program" and 28.passed
When Admin sends "GET" Request and request Body with endpoint for Program...passed
Then Admin receives 401 "Unauthorized" Status with response body for Program.passed
]]></system-out>
</testcase>
<testcase classname="POST REQUEST for Program Module" name="Check if Admin able to create a program with &quot;&lt;scenarioName&gt;&quot; - Example #1.24" time="0.351">
<system-out><![CDATA[
Given User is Authorized....................................................passed
Given Admin creates "GET" Request for the LMS with request body "Program" and 29.passed
When Admin sends "GET" Request and request Body with endpoint for Program...passed
Then Admin receives 404 "Not Found" Status with response body for Program...passed
]]></system-out>
</testcase>
<testcase classname="POST REQUEST for Program Module" name="Check if Admin able to create a program with &quot;&lt;scenarioName&gt;&quot; - Example #1.25" time="0.37">
<system-out><![CDATA[
Given User is Authorized....................................................passed
Given Admin creates "GET" Request for the LMS with request body "Program" and 30.passed
When Admin sends "GET" Request and request Body with endpoint for Program...passed
Then Admin receives 401 "Unauthorized" Status with response body for Program.passed
]]></system-out>
</testcase>
<testcase classname="POST REQUEST for Program Module" name="Check if Admin able to create a program with &quot;&lt;scenarioName&gt;&quot; - Example #1.26" time="0.425">
<system-out><![CDATA[
Given User is Authorized....................................................passed
Given Admin creates "POST" Request for the LMS with request body "Program" and 31.passed
When Admin sends "POST" Request and request Body with endpoint for Program..passed
Then Admin receives 405 "Method Not Allowed" Status with response body for Program.passed
]]></system-out>
</testcase>
<testcase classname="POST REQUEST for Program Module" name="Check if Admin able to create a program with &quot;&lt;scenarioName&gt;&quot; - Example #1.27" time="0.307">
<system-out><![CDATA[
Given User is Authorized....................................................passed
Given Admin creates "POST" Request for the LMS with request body "Program" and 32.passed
When Admin sends "POST" Request and request Body with endpoint for Program..passed
Then Admin receives 405 "Method Not Allowed" Status with response body for Program.passed
]]></system-out>
</testcase>
<testcase classname="POST REQUEST for Program Module" name="Check if Admin able to create a program with &quot;&lt;scenarioName&gt;&quot; - Example #1.28" time="0.317">
<system-out><![CDATA[
Given User is Authorized....................................................passed
Given Admin creates "POST" Request for the LMS with request body "Program" and 33.passed
When Admin sends "POST" Request and request Body with endpoint for Program..passed
Then Admin receives 405 "Method Not Allowed" Status with response body for Program.passed
]]></system-out>
</testcase>
<testcase classname="POST REQUEST for Program Module" name="Check if Admin able to create a program with &quot;&lt;scenarioName&gt;&quot; - Example #1.29" time="0.419">
<system-out><![CDATA[
Given User is Authorized....................................................passed
Given Admin creates "GET" Request for the LMS with request body "Program" and 34.passed
When Admin sends "GET" Request and request Body with endpoint for Program...passed
Then Admin receives 404 "Not Found" Status with response body for Program...passed
]]></system-out>
</testcase>
<testcase classname="POST REQUEST for Program Module" name="Check if Admin able to create a program with &quot;&lt;scenarioName&gt;&quot; - Example #1.30" time="1.599">
<system-out><![CDATA[
Given User is Authorized....................................................passed
Given Admin creates "DELETE" Request for the LMS with request body "Program" and 35.passed
When Admin sends "DELETE" Request and request Body with endpoint for Program.passed
Then Admin receives 200 "Ok" Status with response body for Program..........passed
]]></system-out>
</testcase>
<testcase classname="POST REQUEST for Program Module" name="Check if Admin able to create a program with &quot;&lt;scenarioName&gt;&quot; - Example #1.31" time="0.485">
<system-out><![CDATA[
Given User is Authorized....................................................passed
Given Admin creates "DELETE" Request for the LMS with request body "Program" and 36.passed
When Admin sends "DELETE" Request and request Body with endpoint for Program.passed
Then Admin receives 200 "Ok" Status with response body for Program..........passed
]]></system-out>
</testcase>
<testcase classname="POST REQUEST for Program Module" name="Check if Admin able to create a program with &quot;&lt;scenarioName&gt;&quot; - Example #1.32" time="0.722">
<system-out><![CDATA[
Given User is Authorized....................................................passed
Given Admin creates "DELETE" Request for the LMS with request body "Program" and 37.passed
When Admin sends "DELETE" Request and request Body with endpoint for Program.passed
Then Admin receives 404 "Not Found" Status with response body for Program...passed
]]></system-out>
</testcase>
<testcase classname="POST REQUEST for Program Module" name="Check if Admin able to create a program with &quot;&lt;scenarioName&gt;&quot; - Example #1.33" time="0.825">
<system-out><![CDATA[
Given User is Authorized....................................................passed
Given Admin creates "DELETE" Request for the LMS with request body "Program" and 38.passed
When Admin sends "DELETE" Request and request Body with endpoint for Program.passed
Then Admin receives 404 "Not Found" Status with response body for Program...passed
]]></system-out>
</testcase>
<testcase classname="POST REQUEST for Program Module" name="Check if Admin able to create a program with &quot;&lt;scenarioName&gt;&quot; - Example #1.34" time="0.502">
<system-out><![CDATA[
Given User is Authorized....................................................passed
Given Admin creates "DELETE" Request for the LMS with request body "Program" and 39.passed
When Admin sends "DELETE" Request and request Body with endpoint for Program.passed
Then Admin receives 401 "Unauthorized" Status with response body for Program.passed
]]></system-out>
</testcase>
<testcase classname="POST REQUEST for Program Module" name="Check if Admin able to create a program with &quot;&lt;scenarioName&gt;&quot; - Example #1.35" time="0.601">
<system-out><![CDATA[
Given User is Authorized....................................................passed
Given Admin creates "DELETE" Request for the LMS with request body "Program" and 40.passed
When Admin sends "DELETE" Request and request Body with endpoint for Program.passed
Then Admin receives 401 "Unauthorized" Status with response body for Program.passed
]]></system-out>
</testcase>
<testcase classname="POST REQUEST for Program Module" name="Check if Admin able to create a program with &quot;&lt;scenarioName&gt;&quot; - Example #1.36" time="0.53">
<system-out><![CDATA[
Given User is Authorized....................................................passed
Given Admin creates "DELETE" Request for the LMS with request body "Program" and 41.passed
When Admin sends "DELETE" Request and request Body with endpoint for Program.passed
Then Admin receives 404 "Not Found" Status with response body for Program...passed
]]></system-out>
</testcase>
<testcase classname="POST REQUEST for Program Module" name="Check if Admin able to create a program with &quot;&lt;scenarioName&gt;&quot; - Example #1.37" time="0.314">
<system-out><![CDATA[
Given User is Authorized....................................................passed
Given Admin creates "DELETE" Request for the LMS with request body "Program" and 42.passed
When Admin sends "DELETE" Request and request Body with endpoint for Program.passed
Then Admin receives 404 "Not Found" Status with response body for Program...passed
]]></system-out>
</testcase>
<testcase classname="POST REQUEST for Program Module" name="Check if Admin able to retrieve a program with invalid baseURI - Example #1.1" time="0.91">
<system-out><![CDATA[
Given User is Authorized....................................................passed
Given Admin creates "GET" Request for the LMS API with programID............passed
When Admin sends "GET" Request with invalid BaseURI for Program "https://asdf.com/programs/".passed
Then Admin receives 404 "not found" Status with response body for Program for invalid base url.passed
]]></system-out>
</testcase>
<testcase classname="POST REQUEST for Program Module" name="Check if Admin able to retrieve a program with invalid baseURI - Example #1.2" time="0.431">
<system-out><![CDATA[
Given User is Authorized....................................................passed
Given Admin creates "PUT" Request for the LMS API with programID............passed
When Admin sends "PUT" Request with invalid BaseURI for Program "https://asdf.com//putprogram/".passed
Then Admin receives 404 "not found" Status with response body for Program for invalid base url.passed
]]></system-out>
</testcase>
<testcase classname="POST REQUEST for Program Module" name="Check if Admin able to create a program with &quot;&lt;scenarioName&gt;&quot; - Example #1.1" time="1.012">
<system-out><![CDATA[
Given User is Authorized....................................................passed
Given Admin creates "POST" Request for the LMS with request body "Program" and 1.passed
When Admin sends "POST" Request and request Body with endpoint for Program..passed
Then Admin receives 201 "Created" Status with response body for Program.....passed
Then Admin receives 400 "Bad Request" Status with response body for Program.passed
]]></system-out>
</testcase>
<testcase classname="Post Request Program Batch module" name="Create Batches with external excel data" time="0.314">
<failure type="java.lang.NullPointerException" message="Cannot invoke &quot;org.apache.poi.xssf.usermodel.XSSFRow.getCell(int)&quot; because the return value of &quot;org.apache.poi.xssf.usermodel.XSSFSheet.getRow(int)&quot; is null">
<![CDATA[java.lang.NullPointerException: Cannot invoke "org.apache.poi.xssf.usermodel.XSSFRow.getCell(int)" because the return value of "org.apache.poi.xssf.usermodel.XSSFSheet.getRow(int)" is null
	at Utilities.Batchdatareader.batch(Batchdatareader.java:35)
	at stepDefinations.Batch.sends_http_post_batch_request_with_endpoints_and_read_data_from_external_file(Batch.java:54)
	at ✽.Sends HTTP Post batch request with endpoints and Read data from external file(file:///C:/Users/User/Desktop/Git%20Project%201st%20April%20part2/RestAssured/src/test/resources/Features/C_programbatch.feature:8)
]]>
</failure>
<system-out><![CDATA[
Given Authorized with bearer Token..........................................passed
When Sends HTTP Post batch request with endpoints and Read data from external file.failed
Then Gets 201 status code with response body................................skipped
]]></system-out>
</testcase>
<testcase classname="Post Request Program Batch module" name="Create Batches with invalid token" time="0.15">
<failure type="java.lang.NullPointerException" message="Cannot invoke &quot;org.apache.poi.xssf.usermodel.XSSFRow.getCell(int)&quot; because the return value of &quot;org.apache.poi.xssf.usermodel.XSSFSheet.getRow(int)&quot; is null">
<![CDATA[java.lang.NullPointerException: Cannot invoke "org.apache.poi.xssf.usermodel.XSSFRow.getCell(int)" because the return value of "org.apache.poi.xssf.usermodel.XSSFSheet.getRow(int)" is null
	at Utilities.Batchdatareader.batch(Batchdatareader.java:35)
	at stepDefinations.Batch.sends_http_post_batch_request_with_endpoints(Batch.java:84)
	at ✽.Sends Http Post batch request with endpoints(file:///C:/Users/User/Desktop/Git%20Project%201st%20April%20part2/RestAssured/src/test/resources/Features/C_programbatch.feature:13)
]]>
</failure>
<system-out><![CDATA[
Given Token is expired and invalid..........................................passed
When Sends Http Post batch request with endpoints...........................failed
Then gets 401 unauthorised status code with error message...................skipped
]]></system-out>
</testcase>
<testcase classname="Post Request Program Batch module" name="Create batches with invalid endpoint" time="0.265">
<system-out><![CDATA[
Given Authorized with bearer Token "Invalid endpoint".......................passed
When Sends HTTP Post batch request  with invalid endpoint and Read data "API_hackathon_march24 " "SDETd5 " 5 "active " 16306.passed
Then Gets  404 with response body...........................................passed
]]></system-out>
</testcase>
<testcase classname="Post Request Program Batch module" name="create batches with different examples - Example #1.1" time="0.349">
<failure type="java.lang.AssertionError" message=" Expected status code returned expected [201] but found [404]">
<![CDATA[java.lang.AssertionError:  Expected status code returned expected [201] but found [404]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1578)
	at org.testng.Assert.assertEqualsImpl(Assert.java:150)
	at org.testng.Assert.assertEquals(Assert.java:132)
	at org.testng.Assert.assertEquals(Assert.java:1419)
	at org.testng.Assert.assertEquals(Assert.java:1407)
	at stepDefinations.Batch.gets_with_batch_response_body(Batch.java:173)
	at ✽.Gets  201 with batch response body(file:///C:/Users/User/Desktop/Git%20Project%201st%20April%20part2/RestAssured/src/test/resources/Features/C_programbatch.feature:24)
]]>
</failure>
<system-out><![CDATA[
Given Authorized with bearer Token "Create_batch_with_new_data".............passed
When Sends HTTP Post batch request with valid endpoints and Read data "API_hackathon_march24" "SDETbatchess1" 4 "active" 16306.passed
Then Gets  201 with batch response body.....................................failed
]]></system-out>
</testcase>
<testcase classname="Post Request Program Batch module" name="create batches with different examples - Example #1.2" time="0.239">
<failure type="java.lang.AssertionError" message=" Expected status code returned expected [400] but found [404]">
<![CDATA[java.lang.AssertionError:  Expected status code returned expected [400] but found [404]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1578)
	at org.testng.Assert.assertEqualsImpl(Assert.java:150)
	at org.testng.Assert.assertEquals(Assert.java:132)
	at org.testng.Assert.assertEquals(Assert.java:1419)
	at org.testng.Assert.assertEquals(Assert.java:1407)
	at stepDefinations.Batch.gets_with_batch_response_body(Batch.java:173)
	at ✽.Gets  400 with batch response body(file:///C:/Users/User/Desktop/Git%20Project%201st%20April%20part2/RestAssured/src/test/resources/Features/C_programbatch.feature:24)
]]>
</failure>
<system-out><![CDATA[
Given Authorized with bearer Token "Create_batch_with_existing_data"........passed
When Sends HTTP Post batch request with valid endpoints and Read data "API_hackathon_march24" "SDETbatchess1" 4 "active" 16306.passed
Then Gets  400 with batch response body.....................................failed
]]></system-out>
</testcase>
<testcase classname="Post Request Program Batch module" name="create batches with different examples - Example #1.3" time="0.409">
<failure type="java.lang.AssertionError" message=" Expected status code returned expected [400] but found [404]">
<![CDATA[java.lang.AssertionError:  Expected status code returned expected [400] but found [404]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1578)
	at org.testng.Assert.assertEqualsImpl(Assert.java:150)
	at org.testng.Assert.assertEquals(Assert.java:132)
	at org.testng.Assert.assertEquals(Assert.java:1419)
	at org.testng.Assert.assertEquals(Assert.java:1407)
	at stepDefinations.Batch.gets_with_batch_response_body(Batch.java:173)
	at ✽.Gets  400 with batch response body(file:///C:/Users/User/Desktop/Git%20Project%201st%20April%20part2/RestAssured/src/test/resources/Features/C_programbatch.feature:24)
]]>
</failure>
<system-out><![CDATA[
Given Authorized with bearer Token "Post_missing_mandatory_fields"..........passed
When Sends HTTP Post batch request with valid endpoints and Read data "API_hackathon_march24" "" 4 "active" 16306.passed
Then Gets  400 with batch response body.....................................failed
]]></system-out>
</testcase>
<testcase classname="Post Request Program Batch module" name="create batches with different examples - Example #1.4" time="0.328">
<failure type="java.lang.AssertionError" message=" Expected status code returned expected [201] but found [404]">
<![CDATA[java.lang.AssertionError:  Expected status code returned expected [201] but found [404]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1578)
	at org.testng.Assert.assertEqualsImpl(Assert.java:150)
	at org.testng.Assert.assertEquals(Assert.java:132)
	at org.testng.Assert.assertEquals(Assert.java:1419)
	at org.testng.Assert.assertEquals(Assert.java:1407)
	at stepDefinations.Batch.gets_with_batch_response_body(Batch.java:173)
	at ✽.Gets  201 with batch response body(file:///C:/Users/User/Desktop/Git%20Project%201st%20April%20part2/RestAssured/src/test/resources/Features/C_programbatch.feature:24)
]]>
</failure>
<system-out><![CDATA[
Given Authorized with bearer Token "Post_missing_additional_fields_null"....passed
When Sends HTTP Post batch request with valid endpoints and Read data "null" "SDETrestss1" 4 "active" 16306.passed
Then Gets  201 with batch response body.....................................failed
]]></system-out>
</testcase>
<testcase classname="Post Request Program Batch module" name="create batches with different examples - Example #1.5" time="0.233">
<failure type="java.lang.AssertionError" message=" Expected status code returned expected [400] but found [404]">
<![CDATA[java.lang.AssertionError:  Expected status code returned expected [400] but found [404]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1578)
	at org.testng.Assert.assertEqualsImpl(Assert.java:150)
	at org.testng.Assert.assertEquals(Assert.java:132)
	at org.testng.Assert.assertEquals(Assert.java:1419)
	at org.testng.Assert.assertEquals(Assert.java:1407)
	at stepDefinations.Batch.gets_with_batch_response_body(Batch.java:173)
	at ✽.Gets  400 with batch response body(file:///C:/Users/User/Desktop/Git%20Project%201st%20April%20part2/RestAssured/src/test/resources/Features/C_programbatch.feature:24)
]]>
</failure>
<system-out><![CDATA[
Given Authorized with bearer Token "Post_missing_additional_fields_null"....passed
When Sends HTTP Post batch request with valid endpoints and Read data "" "SDETapi" 4 "active" 16306.passed
Then Gets  400 with batch response body.....................................failed
]]></system-out>
</testcase>
<testcase classname="Post Request Program Batch module" name="create batches with different examples - Example #1.6" time="0.216">
<failure type="java.lang.AssertionError" message=" Expected status code returned expected [400] but found [404]">
<![CDATA[java.lang.AssertionError:  Expected status code returned expected [400] but found [404]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1578)
	at org.testng.Assert.assertEqualsImpl(Assert.java:150)
	at org.testng.Assert.assertEquals(Assert.java:132)
	at org.testng.Assert.assertEquals(Assert.java:1419)
	at org.testng.Assert.assertEquals(Assert.java:1407)
	at stepDefinations.Batch.gets_with_batch_response_body(Batch.java:173)
	at ✽.Gets  400 with batch response body(file:///C:/Users/User/Desktop/Git%20Project%201st%20April%20part2/RestAssured/src/test/resources/Features/C_programbatch.feature:24)
]]>
</failure>
<system-out><![CDATA[
Given Authorized with bearer Token "Invalid_Progam_Id"......................passed
When Sends HTTP Post batch request with valid endpoints and Read data "API_hackathon_march24" "SDET" 4 "active" 16900.passed
Then Gets  400 with batch response body.....................................failed
]]></system-out>
</testcase>
<testcase classname="Post Request Program Batch module" name="create batches with different examples - Example #1.7" time="0.257">
<failure type="java.lang.AssertionError" message=" Expected status code returned expected [400] but found [404]">
<![CDATA[java.lang.AssertionError:  Expected status code returned expected [400] but found [404]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1578)
	at org.testng.Assert.assertEqualsImpl(Assert.java:150)
	at org.testng.Assert.assertEquals(Assert.java:132)
	at org.testng.Assert.assertEquals(Assert.java:1419)
	at org.testng.Assert.assertEquals(Assert.java:1407)
	at stepDefinations.Batch.gets_with_batch_response_body(Batch.java:173)
	at ✽.Gets  400 with batch response body(file:///C:/Users/User/Desktop/Git%20Project%201st%20April%20part2/RestAssured/src/test/resources/Features/C_programbatch.feature:24)
]]>
</failure>
<system-out><![CDATA[
Given Authorized with bearer Token "Post_invalid_data"......................passed
When Sends HTTP Post batch request with valid endpoints and Read data "API_hackathon_march24" "SDET" 4 "active" 16306.passed
Then Gets  400 with batch response body.....................................failed
]]></system-out>
</testcase>
<testcase classname="GET Request Program Batch module" name="Get all batches" time="0.297">
<failure type="java.lang.AssertionError" message="Expected status code returned expected [200] but found [404]">
<![CDATA[java.lang.AssertionError: Expected status code returned expected [200] but found [404]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1578)
	at org.testng.Assert.assertEqualsImpl(Assert.java:150)
	at org.testng.Assert.assertEquals(Assert.java:132)
	at org.testng.Assert.assertEquals(Assert.java:1419)
	at org.testng.Assert.assertEquals(Assert.java:1407)
	at stepDefinations.getBatch.gets_status_code_for_get_batch_with_response_body(getBatch.java:55)
	at ✽.Gets 200 status code for get batch with response body(file:///C:/Users/User/Desktop/Git%20Project%201st%20April%20part2/RestAssured/src/test/resources/Features/D_getbatch.feature:9)
]]>
</failure>
<system-out><![CDATA[
Given GET Batches Authorized with bearer Token..............................passed
When Sends HTTP GET batch request with valid endpoints......................passed
Then Gets 200 status code for get batch with response body..................failed
]]></system-out>
</testcase>
<testcase classname="GET Request Program Batch module" name="Get all batches as Unauthorized" time="0.409">
<system-out><![CDATA[
Given Get batch token is invalid............................................passed
When Sends HTTP GET batch request with valid endpoints......................passed
Then Gets 401 status code for get batch with unauthorised message...........passed
]]></system-out>
</testcase>
<testcase classname="GET Request Program Batch module" name="Get all batches with invalid enpoints" time="0.219">
<system-out><![CDATA[
Given GET Batches Authorized with bearer Token..............................passed
When Sends HTTP GET batch request with invalid endpoints....................passed
Then Gets 404 status for get batch with error message Not Found.............passed
]]></system-out>
</testcase>
<testcase classname="GET Request Program Batch module" name="Get all batches with search string" time="0.367">
<failure type="java.lang.AssertionError" message="Expected status code returned expected [200] but found [404]">
<![CDATA[java.lang.AssertionError: Expected status code returned expected [200] but found [404]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1578)
	at org.testng.Assert.assertEqualsImpl(Assert.java:150)
	at org.testng.Assert.assertEquals(Assert.java:132)
	at org.testng.Assert.assertEquals(Assert.java:1419)
	at org.testng.Assert.assertEquals(Assert.java:1407)
	at stepDefinations.getBatch.gets_status_for_get_batch_with_response_body(getBatch.java:126)
	at ✽.Gets 200 status for get batch with response body(file:///C:/Users/User/Desktop/Git%20Project%201st%20April%20part2/RestAssured/src/test/resources/Features/D_getbatch.feature:24)
]]>
</failure>
<system-out><![CDATA[
Given GET Batches Authorized with bearer Token..............................passed
When Sends HTTP GET batch request with search endpoints.....................passed
Then Gets 200 status for get batch with response body.......................failed
]]></system-out>
</testcase>
<testcase classname="GET Request Program Batch module" name="Get all batches with search parameter with value" time="1.054">
<failure type="java.lang.AssertionError" message="Expected status code returned expected [200] but found [404]">
<![CDATA[java.lang.AssertionError: Expected status code returned expected [200] but found [404]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1578)
	at org.testng.Assert.assertEqualsImpl(Assert.java:150)
	at org.testng.Assert.assertEquals(Assert.java:132)
	at org.testng.Assert.assertEquals(Assert.java:1419)
	at org.testng.Assert.assertEquals(Assert.java:1407)
	at stepDefinations.getBatch.gets_status_for_get_batch_with_response_body(getBatch.java:126)
	at ✽.Gets 200 status for get batch with response body(file:///C:/Users/User/Desktop/Git%20Project%201st%20April%20part2/RestAssured/src/test/resources/Features/D_getbatch.feature:29)
]]>
</failure>
<system-out><![CDATA[
Given GET Batches Authorized with bearer Token..............................passed
When Sends HTTP GET batch request with parameter and value..................passed
Then Gets 200 status for get batch with response body.......................failed
]]></system-out>
</testcase>
<testcase classname="GET BatchId Request Program Batch module" name="Get Batches with BatchId" time="0.468">
<failure type="java.lang.AssertionError" message="Expected status code returned expected [200] but found [404]">
<![CDATA[java.lang.AssertionError: Expected status code returned expected [200] but found [404]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1578)
	at org.testng.Assert.assertEqualsImpl(Assert.java:150)
	at org.testng.Assert.assertEquals(Assert.java:132)
	at org.testng.Assert.assertEquals(Assert.java:1419)
	at org.testng.Assert.assertEquals(Assert.java:1407)
	at stepDefinations.getbatchid.gets_status_code_for_get_batch_id_with_response_body(getbatchid.java:51)
	at ✽.Gets 200 status code for get batchID with response body(file:///C:/Users/User/Desktop/Git%20Project%201st%20April%20part2/RestAssured/src/test/resources/Features/E_getbatchid.feature:9)
]]>
</failure>
<system-out><![CDATA[
Given GET BatchId Authorized with bearer Token..............................passed
When Sends HTTP GET batch request with BatchId valid endpoints..............passed
Then Gets 200 status code for get batchID with response body................failed
]]></system-out>
</testcase>
<testcase classname="GET BatchId Request Program Batch module" name="Get Batches with BatchId as Unauthorized" time="0.26">
<system-out><![CDATA[
Given Get batchId token is invalid..........................................passed
When Sends HTTP GET batch requests  with BatchId valid endpoints............passed
Then Gets 401 status code for get batchID with unauthorised message.........passed
]]></system-out>
</testcase>
<testcase classname="GET BatchId Request Program Batch module" name="Get Batches with BatchId with invalid enpoints" time="0.263">
<failure type="java.lang.AssertionError" message="Expected status code returned expected [405] but found [404]">
<![CDATA[java.lang.AssertionError: Expected status code returned expected [405] but found [404]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1578)
	at org.testng.Assert.assertEqualsImpl(Assert.java:150)
	at org.testng.Assert.assertEquals(Assert.java:132)
	at org.testng.Assert.assertEquals(Assert.java:1419)
	at org.testng.Assert.assertEquals(Assert.java:1407)
	at stepDefinations.getbatchid.gets_status_for_get_batch_id_with_error_message(getbatchid.java:113)
	at ✽.Gets 405 status for get batchID with error message(file:///C:/Users/User/Desktop/Git%20Project%201st%20April%20part2/RestAssured/src/test/resources/Features/E_getbatchid.feature:19)
]]>
</failure>
<system-out><![CDATA[
Given GET BatchId Authorized with bearer Token..............................passed
When Sends HTTP GET batch request with BatchId invalid endpoints............passed
Then Gets 405 status for get batchID with error message.....................failed
]]></system-out>
</testcase>
<testcase classname="GET BatchId Request Program Batch module" name="Get Batches with Invalid BatchId" time="0.335">
<failure type="java.lang.AssertionError" message="expected [true] but found [false]">
<![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1578)
	at org.testng.Assert.assertTrue(Assert.java:57)
	at org.testng.Assert.assertTrue(Assert.java:67)
	at stepDefinations.getbatchid.gets_status_for_get_batch_id_with_error_message(getbatchid.java:121)
	at ✽.Gets 404 status for get batchID with error message(file:///C:/Users/User/Desktop/Git%20Project%201st%20April%20part2/RestAssured/src/test/resources/Features/E_getbatchid.feature:24)
]]>
</failure>
<system-out><![CDATA[
Given GET BatchId Authorized with bearer Token..............................passed
When Sends HTTP GET batch request with invalid BatchId and valid endpoints..passed
Then Gets 404 status for get batchID with error message.....................failed
]]></system-out>
</testcase>
<testcase classname="GET BatchId Request Program Batch module" name="Get Batches with BatchId with search parameter with value" time="0.221">
<failure type="java.lang.AssertionError" message="Expected status code returned expected [200] but found [404]">
<![CDATA[java.lang.AssertionError: Expected status code returned expected [200] but found [404]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1578)
	at org.testng.Assert.assertEqualsImpl(Assert.java:150)
	at org.testng.Assert.assertEquals(Assert.java:132)
	at org.testng.Assert.assertEquals(Assert.java:1419)
	at org.testng.Assert.assertEquals(Assert.java:1407)
	at stepDefinations.getbatchid.gets_status_for_get_batch_id_with_response_body(getbatchid.java:153)
	at ✽.Gets 200 status for get batchID with response body(file:///C:/Users/User/Desktop/Git%20Project%201st%20April%20part2/RestAssured/src/test/resources/Features/E_getbatchid.feature:29)
]]>
</failure>
<system-out><![CDATA[
Given GET BatchId Authorized with bearer Token..............................passed
When Sends HTTP GET batch request with BatchId parameter and value..........passed
Then Gets 200 status for get batchID with response body.....................failed
]]></system-out>
</testcase>
<testcase classname="GET BatchName Request Program Batch module" name="Get Batches with BatchName" time="0.399">
<failure type="java.lang.AssertionError" message="Expected status code returned expected [200] but found [404]">
<![CDATA[java.lang.AssertionError: Expected status code returned expected [200] but found [404]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1578)
	at org.testng.Assert.assertEqualsImpl(Assert.java:150)
	at org.testng.Assert.assertEquals(Assert.java:132)
	at org.testng.Assert.assertEquals(Assert.java:1419)
	at org.testng.Assert.assertEquals(Assert.java:1407)
	at stepDefinations.getbatchname.gets_status_code_for_get_batch_name_with_response_body(getbatchname.java:50)
	at ✽.Gets 200 status code for get BatchName with response body(file:///C:/Users/User/Desktop/Git%20Project%201st%20April%20part2/RestAssured/src/test/resources/Features/F_getbatchname.feature:9)
]]>
</failure>
<system-out><![CDATA[
Given GET BatchName Authorized with bearer Token............................passed
When Sends HTTP GET batch request with BatchName valid endpoints............passed
Then Gets 200 status code for get BatchName with response body..............failed
]]></system-out>
</testcase>
<testcase classname="GET BatchName Request Program Batch module" name="Get Batches with BatchName as Unauthorized" time="0.344">
<system-out><![CDATA[
Given Get BatchName token is invalid........................................passed
When Sends HTTP GET batch requests  with BatchName valid endpoints..........passed
Then Gets 401 status code for get BatchName with unauthorised message.......passed
]]></system-out>
</testcase>
<testcase classname="GET BatchName Request Program Batch module" name="Get Batches with BatchName with invalid enpoints" time="0.266">
<failure type="java.lang.AssertionError" message="Expected status code returned expected [405] but found [404]">
<![CDATA[java.lang.AssertionError: Expected status code returned expected [405] but found [404]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1578)
	at org.testng.Assert.assertEqualsImpl(Assert.java:150)
	at org.testng.Assert.assertEquals(Assert.java:132)
	at org.testng.Assert.assertEquals(Assert.java:1419)
	at org.testng.Assert.assertEquals(Assert.java:1407)
	at stepDefinations.getbatchname.gets_status_for_get_batch_name_with_error_message(getbatchname.java:111)
	at ✽.Gets 405 status for get BatchName with error message(file:///C:/Users/User/Desktop/Git%20Project%201st%20April%20part2/RestAssured/src/test/resources/Features/F_getbatchname.feature:19)
]]>
</failure>
<system-out><![CDATA[
Given GET BatchName Authorized with bearer Token............................passed
When Sends HTTP GET batch request with BatchName invalid endpoints..........passed
Then Gets 405 status for get BatchName with error message...................failed
]]></system-out>
</testcase>
<testcase classname="GET BatchName Request Program Batch module" name="Get Batches with Invalid BatchName" time="0.258">
<failure type="java.lang.AssertionError" message="expected [true] but found [false]">
<![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1578)
	at org.testng.Assert.assertTrue(Assert.java:57)
	at org.testng.Assert.assertTrue(Assert.java:67)
	at stepDefinations.getbatchname.gets_status_for_get_batch_name_with_error_message(getbatchname.java:120)
	at ✽.Gets 404 status for get BatchName with error message(file:///C:/Users/User/Desktop/Git%20Project%201st%20April%20part2/RestAssured/src/test/resources/Features/F_getbatchname.feature:24)
]]>
</failure>
<system-out><![CDATA[
Given GET BatchName Authorized with bearer Token............................passed
When Sends HTTP GET batch request with invalid BatchName and valid endpoints.passed
Then Gets 404 status for get BatchName with error message...................failed
]]></system-out>
</testcase>
<testcase classname="GET BatchName Request Program Batch module" name="Get Batches with BatchName with BatchId value as a parameter" time="0.253">
<failure type="java.lang.AssertionError" message="Expected status code returned expected [200] but found [404]">
<![CDATA[java.lang.AssertionError: Expected status code returned expected [200] but found [404]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1578)
	at org.testng.Assert.assertEqualsImpl(Assert.java:150)
	at org.testng.Assert.assertEquals(Assert.java:132)
	at org.testng.Assert.assertEquals(Assert.java:1419)
	at org.testng.Assert.assertEquals(Assert.java:1407)
	at stepDefinations.getbatchname.gets_status_for_get_batch_name_with_response_body(getbatchname.java:154)
	at ✽.Gets 200 status for get BatchName with response body(file:///C:/Users/User/Desktop/Git%20Project%201st%20April%20part2/RestAssured/src/test/resources/Features/F_getbatchname.feature:29)
]]>
</failure>
<system-out><![CDATA[
Given GET BatchName Authorized with bearer Token............................passed
When Sends HTTP GET batch request with BatchName parameter and value........passed
Then Gets 200 status for get BatchName with response body...................failed
]]></system-out>
</testcase>
<testcase classname="Get Request for ProgramBatch Module" name="Check if admin able to retrieve a batch without authorization - Example #1.1" time="0.28">
<system-out><![CDATA[
Given Admin creates GET Request with program id and unAuth..................passed
When Admin sends HTTPS Request with endpoint and unAuth.....................passed
Then Admin receives Status with 401 and statusmessage unauthorized..........passed
]]></system-out>
</testcase>
<testcase classname="Get Request for ProgramBatch Module" name="Check if admin able to retrieve a batch with valid Program ID - Example #1.1" time="2.167">
<failure type="java.lang.AssertionError" message="1 expectation failed.
Response body doesn't match expectation.
Expected: The content to match the given JSON schema.
error: instance type (object) does not match any allowed primitive type (allowed: [&quot;array&quot;])
    level: &quot;error&quot;
    schema: {&quot;loadingURI&quot;:&quot;file:/C:/Users/User/Desktop/Git%20Project%201st%20April%20part2/RestAssured/target/test-classes/batchProgramId.json#&quot;,&quot;pointer&quot;:&quot;&quot;}
    instance: {&quot;pointer&quot;:&quot;&quot;}
    domain: &quot;validation&quot;
    keyword: &quot;type&quot;
    found: &quot;object&quot;
    expected: [&quot;array&quot;]

  Actual: {&quot;timestamp&quot;:&quot;2024-04-02T00:59:54.817+00:00&quot;,&quot;status&quot;:404,&quot;error&quot;:&quot;Not Found&quot;,&quot;path&quot;:&quot;/lms/batches/program/16306/saveprogram&quot;}
">
<![CDATA[java.lang.AssertionError: 1 expectation failed.
Response body doesn't match expectation.
Expected: The content to match the given JSON schema.
error: instance type (object) does not match any allowed primitive type (allowed: ["array"])
    level: "error"
    schema: {"loadingURI":"file:/C:/Users/User/Desktop/Git%20Project%201st%20April%20part2/RestAssured/target/test-classes/batchProgramId.json#","pointer":""}
    instance: {"pointer":""}
    domain: "validation"
    keyword: "type"
    found: "object"
    expected: ["array"]

  Actual: {"timestamp":"2024-04-02T00:59:54.817+00:00","status":404,"error":"Not Found","path":"/lms/batches/program/16306/saveprogram"}

	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
	at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:73)
	at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:108)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:57)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:263)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:512)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
	at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:696)
	at io.restassured.internal.ResponseSpecificationImpl.this$2$validateResponseIfRequired(ResponseSpecificationImpl.groovy)
	at jdk.internal.reflect.GeneratedMethodAccessor259.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:198)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:62)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:49)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:171)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)
	at io.restassured.internal.ResponseSpecificationImpl.body(ResponseSpecificationImpl.groovy:107)
	at io.restassured.internal.ValidatableResponseOptionsImpl.body(ValidatableResponseOptionsImpl.java:238)
	at stepDefinations.Get_Batch_ProgramId.admin_receives_ok_status_with_response_body(Get_Batch_ProgramId.java:68)
	at ✽.Admin receives 200 OK Status with response body.(file:///C:/Users/User/Desktop/Git%20Project%201st%20April%20part2/RestAssured/src/test/resources/Features/G_batch_GetProgramId.feature:19)
]]>
</failure>
<system-out><![CDATA[
Given Admin creates GET Request with valid ProgramId........................passed
When Admin sends HTTPS Request with endpoint with programId.................passed
Then Admin receives 200 OK Status with response body........................failed
]]></system-out>
</testcase>
<testcase classname="Get Request for ProgramBatch Module" name="Check if admin able to retrieve a batch with invalid Program Id - Example #1.1" time="0.362">
<system-out><![CDATA[
Given Admin creates GET Request with invalid Program Id.....................passed
When Admin sends HTTPS Request with endpoint with invalid programId.........passed
Then Admin receives 404 and Not Found Status with message and boolean success details.passed
]]></system-out>
</testcase>
<testcase classname="Get Request for ProgramBatch Module" name="Check if admin able to retrieve a batch with invalid endpoint - Example #1.1" time="0.316">
<failure type="java.lang.AssertionError" message="1 expectation failed.
Expected status code &lt;400&gt; but was &lt;404&gt;.
">
<![CDATA[java.lang.AssertionError: 1 expectation failed.
Expected status code <400> but was <404>.

	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
	at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:73)
	at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:108)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:512)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
	at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:696)
	at io.restassured.internal.ResponseSpecificationImpl.this$2$validateResponseIfRequired(ResponseSpecificationImpl.groovy)
	at jdk.internal.reflect.GeneratedMethodAccessor259.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:198)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:62)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)
	at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:135)
	at io.restassured.specification.ResponseSpecification$statusCode$0.callCurrent(Unknown Source)
	at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:143)
	at io.restassured.internal.ValidatableResponseOptionsImpl.statusCode(ValidatableResponseOptionsImpl.java:89)
	at stepDefinations.Get_Batch_ProgramId.admin_receives_status_with_error_message_not_found(Get_Batch_ProgramId.java:113)
	at ✽.Admin receives 400  Status with  error message Not Found.(file:///C:/Users/User/Desktop/Git%20Project%201st%20April%20part2/RestAssured/src/test/resources/Features/G_batch_GetProgramId.feature:42)
]]>
</failure>
<system-out><![CDATA[
Given Admin creates GET Request with invalid endpoint.......................passed
When Admin sends HTTPS Request with invalid endpoint........................passed
Then Admin receives 400  Status with  error message Not Found...............failed
]]></system-out>
</testcase>
<testcase classname="Post Request for ProgramBatch Module" name="Check if admin able to create a Batch with valid endpoint and request body (non existing values) - Example #1.1" time="0.886">
<failure type="java.lang.AssertionError" message="1 expectation failed.
Expected status code &lt;201&gt; but was &lt;405&gt;.
">
<![CDATA[java.lang.AssertionError: 1 expectation failed.
Expected status code <201> but was <405>.

	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
	at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:73)
	at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:108)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:512)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
	at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:696)
	at io.restassured.internal.ResponseSpecificationImpl.this$2$validateResponseIfRequired(ResponseSpecificationImpl.groovy)
	at jdk.internal.reflect.GeneratedMethodAccessor259.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:198)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:62)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)
	at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:135)
	at io.restassured.specification.ResponseSpecification$statusCode$0.callCurrent(Unknown Source)
	at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:143)
	at io.restassured.internal.ValidatableResponseOptionsImpl.statusCode(ValidatableResponseOptionsImpl.java:89)
	at stepDefinations.Program_Batch.admin_receives_and_status_with_response_body(Program_Batch.java:88)
	at ✽.Admin receives 201 and  "Created" Status with response body.(file:///C:/Users/User/Desktop/Git%20Project%201st%20April%20part2/RestAssured/src/test/resources/Features/H_program_batch.feature:9)
]]>
</failure>
<system-out><![CDATA[
Given Admin creates POST Request  with valid data in request body "Batch" and 1.passed
When Admin sends HTTPS Request with endpoint................................passed
Then Admin receives 201 and  "Created" Status with response body............failed
]]></system-out>
</testcase>
<testcase classname="Put Request for ProgramBatch Module" name="Check if admin able to update a Batch with valid batchID and mandatory fields in request body - Example #1.1" time="3.741">
<failure type="java.lang.AssertionError" message="1 expectation failed.
Response body doesn't match expectation.
Expected: The content to match the given JSON schema.
error: object has missing required properties ([&quot;batchDescription&quot;,&quot;batchId&quot;,&quot;batchName&quot;,&quot;batchNoOfClasses&quot;,&quot;batchStatus&quot;,&quot;programId&quot;,&quot;programName&quot;])
    level: &quot;error&quot;
    schema: {&quot;loadingURI&quot;:&quot;file:/C:/Users/User/Desktop/Git%20Project%201st%20April%20part2/RestAssured/target/test-classes/batchUpdate.json#&quot;,&quot;pointer&quot;:&quot;&quot;}
    instance: {&quot;pointer&quot;:&quot;&quot;}
    domain: &quot;validation&quot;
    keyword: &quot;required&quot;
    required: [&quot;batchDescription&quot;,&quot;batchId&quot;,&quot;batchName&quot;,&quot;batchNoOfClasses&quot;,&quot;batchStatus&quot;,&quot;programId&quot;,&quot;programName&quot;]
    missing: [&quot;batchDescription&quot;,&quot;batchId&quot;,&quot;batchName&quot;,&quot;batchNoOfClasses&quot;,&quot;batchStatus&quot;,&quot;programId&quot;,&quot;programName&quot;]

  Actual: {&quot;timestamp&quot;:&quot;2024-04-02T00:59:59.860+00:00&quot;,&quot;status&quot;:404,&quot;error&quot;:&quot;Not Found&quot;,&quot;path&quot;:&quot;/lms/batches/0/saveprogram&quot;}
">
<![CDATA[java.lang.AssertionError: 1 expectation failed.
Response body doesn't match expectation.
Expected: The content to match the given JSON schema.
error: object has missing required properties (["batchDescription","batchId","batchName","batchNoOfClasses","batchStatus","programId","programName"])
    level: "error"
    schema: {"loadingURI":"file:/C:/Users/User/Desktop/Git%20Project%201st%20April%20part2/RestAssured/target/test-classes/batchUpdate.json#","pointer":""}
    instance: {"pointer":""}
    domain: "validation"
    keyword: "required"
    required: ["batchDescription","batchId","batchName","batchNoOfClasses","batchStatus","programId","programName"]
    missing: ["batchDescription","batchId","batchName","batchNoOfClasses","batchStatus","programId","programName"]

  Actual: {"timestamp":"2024-04-02T00:59:59.860+00:00","status":404,"error":"Not Found","path":"/lms/batches/0/saveprogram"}

	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
	at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:73)
	at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:108)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:512)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
	at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:696)
	at io.restassured.internal.ResponseSpecificationImpl.this$2$validateResponseIfRequired(ResponseSpecificationImpl.groovy)
	at jdk.internal.reflect.GeneratedMethodAccessor259.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:198)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:62)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)
	at io.restassured.internal.ResponseSpecificationImpl.body(ResponseSpecificationImpl.groovy:107)
	at io.restassured.internal.ValidatableResponseOptionsImpl.body(ValidatableResponseOptionsImpl.java:238)
	at stepDefinations.Batch_Update.admin_receives_and_status_messeage_with_updated_value_in_response_body(Batch_Update.java:104)
	at ✽.Admin receives 200 and  Status messeage with updated value in response body.(file:///C:/Users/User/Desktop/Git%20Project%201st%20April%20part2/RestAssured/src/test/resources/Features/I_batch_update.feature:10)
]]>
</failure>
<system-out><![CDATA[
Given Admin creates PUT Request with the valid BatchId anddata "Batch" and 1.passed
When Admin sends HTTPS Request  with endpoints..............................passed
Then Admin receives 200 and  Status messeage with updated value in response body..failed
]]></system-out>
</testcase>
<testcase classname="Put Request for ProgramBatch Module" name="Check if admin able to update a Batch with &quot;&lt;scenario&gt;&quot; - Example #1.1" time="2.647">
<failure type="java.lang.AssertionError" message="1 expectation failed.
Expected status code &lt;200&gt; but was &lt;404&gt;.
">
<![CDATA[java.lang.AssertionError: 1 expectation failed.
Expected status code <200> but was <404>.

	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
	at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:73)
	at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:108)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:512)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
	at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:696)
	at io.restassured.internal.ResponseSpecificationImpl.this$2$validateResponseIfRequired(ResponseSpecificationImpl.groovy)
	at jdk.internal.reflect.GeneratedMethodAccessor259.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:198)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:62)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)
	at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:135)
	at io.restassured.specification.ResponseSpecification$statusCode$0.callCurrent(Unknown Source)
	at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:143)
	at io.restassured.internal.ValidatableResponseOptionsImpl.statusCode(ValidatableResponseOptionsImpl.java:89)
	at stepDefinations.Batch_Update.admin_receives_statuscode_status_with_the_message_and_boolean_success_details(Batch_Update.java:163)
	at ✽.Admin receives 200 Status with  the message and boolean success details(file:///C:/Users/User/Desktop/Git%20Project%201st%20April%20part2/RestAssured/src/test/resources/Features/I_batch_update.feature:20)
]]>
</failure>
<system-out><![CDATA[
Given Admin creates PUT Request with request body "",3,"active","twosedet137" and "automation".passed
When Admin sends HTTPS Request  with endpoint with negative scenarios.......passed
Then Admin receives 200 Status with  the message and boolean success details.failed
]]></system-out>
</testcase>
<testcase classname="Put Request for ProgramBatch Module" name="Check if admin able to update a Batch with &quot;&lt;scenario&gt;&quot; - Example #1.2" time="0.41">
<failure type="java.lang.AssertionError" message="1 expectation failed.
Expected status code &lt;500&gt; but was &lt;404&gt;.
">
<![CDATA[java.lang.AssertionError: 1 expectation failed.
Expected status code <500> but was <404>.

	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
	at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:73)
	at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:108)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:512)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
	at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:696)
	at io.restassured.internal.ResponseSpecificationImpl.this$2$validateResponseIfRequired(ResponseSpecificationImpl.groovy)
	at jdk.internal.reflect.GeneratedMethodAccessor259.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:198)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:62)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)
	at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:135)
	at io.restassured.specification.ResponseSpecification$statusCode$0.callCurrent(Unknown Source)
	at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:143)
	at io.restassured.internal.ValidatableResponseOptionsImpl.statusCode(ValidatableResponseOptionsImpl.java:89)
	at stepDefinations.Batch_Update.admin_receives_statuscode_status_with_the_message_and_boolean_success_details(Batch_Update.java:163)
	at ✽.Admin receives 500 Status with  the message and boolean success details(file:///C:/Users/User/Desktop/Git%20Project%201st%20April%20part2/RestAssured/src/test/resources/Features/I_batch_update.feature:20)
]]>
</failure>
<system-out><![CDATA[
Given Admin creates PUT Request with request body "description",4,"","twosedet137" and "automation".passed
When Admin sends HTTPS Request  with endpoint with negative scenarios.......passed
Then Admin receives 500 Status with  the message and boolean success details.failed
]]></system-out>
</testcase>
<testcase classname="Put Request for ProgramBatch Module" name="Check if admin able to update a Batch with &quot;&lt;scenario&gt;&quot; - Example #1.3" time="0.302">
<failure type="java.lang.AssertionError" message="1 expectation failed.
Expected status code &lt;500&gt; but was &lt;404&gt;.
">
<![CDATA[java.lang.AssertionError: 1 expectation failed.
Expected status code <500> but was <404>.

	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
	at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:73)
	at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:108)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:512)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
	at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:696)
	at io.restassured.internal.ResponseSpecificationImpl.this$2$validateResponseIfRequired(ResponseSpecificationImpl.groovy)
	at jdk.internal.reflect.GeneratedMethodAccessor259.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:198)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:62)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)
	at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:135)
	at io.restassured.specification.ResponseSpecification$statusCode$0.callCurrent(Unknown Source)
	at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:143)
	at io.restassured.internal.ValidatableResponseOptionsImpl.statusCode(ValidatableResponseOptionsImpl.java:89)
	at stepDefinations.Batch_Update.admin_receives_statuscode_status_with_the_message_and_boolean_success_details(Batch_Update.java:163)
	at ✽.Admin receives 500 Status with  the message and boolean success details(file:///C:/Users/User/Desktop/Git%20Project%201st%20April%20part2/RestAssured/src/test/resources/Features/I_batch_update.feature:20)
]]>
</failure>
<system-out><![CDATA[
Given Admin creates PUT Request with request body "description",1,"active","" and "automation".passed
When Admin sends HTTPS Request  with endpoint with negative scenarios.......passed
Then Admin receives 500 Status with  the message and boolean success details.failed
]]></system-out>
</testcase>
<testcase classname="Put Request for ProgramBatch Module" name="Check if admin able to update a Batch with &quot;&lt;scenario&gt;&quot; - Example #1.4" time="0.346">
<failure type="java.lang.AssertionError" message="1 expectation failed.
Expected status code &lt;500&gt; but was &lt;404&gt;.
">
<![CDATA[java.lang.AssertionError: 1 expectation failed.
Expected status code <500> but was <404>.

	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
	at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:73)
	at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:108)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:512)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
	at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:696)
	at io.restassured.internal.ResponseSpecificationImpl.this$2$validateResponseIfRequired(ResponseSpecificationImpl.groovy)
	at jdk.internal.reflect.GeneratedMethodAccessor259.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:198)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:62)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)
	at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:135)
	at io.restassured.specification.ResponseSpecification$statusCode$0.callCurrent(Unknown Source)
	at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:143)
	at io.restassured.internal.ValidatableResponseOptionsImpl.statusCode(ValidatableResponseOptionsImpl.java:89)
	at stepDefinations.Batch_Update.admin_receives_statuscode_status_with_the_message_and_boolean_success_details(Batch_Update.java:163)
	at ✽.Admin receives 500 Status with  the message and boolean success details(file:///C:/Users/User/Desktop/Git%20Project%201st%20April%20part2/RestAssured/src/test/resources/Features/I_batch_update.feature:20)
]]>
</failure>
<system-out><![CDATA[
Given Admin creates PUT Request with request body "description",2,"active","twosedet137" and "".passed
When Admin sends HTTPS Request  with endpoint with negative scenarios.......passed
Then Admin receives 500 Status with  the message and boolean success details.failed
]]></system-out>
</testcase>
<testcase classname="Put Request for ProgramBatch Module" name="Check if admin able to update a Batch with &quot;&lt;scenario&gt;&quot; - Example #1.5" time="0.3">
<failure type="java.lang.AssertionError" message="1 expectation failed.
Expected status code &lt;500&gt; but was &lt;404&gt;.
">
<![CDATA[java.lang.AssertionError: 1 expectation failed.
Expected status code <500> but was <404>.

	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
	at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:73)
	at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:108)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:512)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
	at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:696)
	at io.restassured.internal.ResponseSpecificationImpl.this$2$validateResponseIfRequired(ResponseSpecificationImpl.groovy)
	at jdk.internal.reflect.GeneratedMethodAccessor259.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:198)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:62)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)
	at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:135)
	at io.restassured.specification.ResponseSpecification$statusCode$0.callCurrent(Unknown Source)
	at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:143)
	at io.restassured.internal.ValidatableResponseOptionsImpl.statusCode(ValidatableResponseOptionsImpl.java:89)
	at stepDefinations.Batch_Update.admin_receives_statuscode_status_with_the_message_and_boolean_success_details(Batch_Update.java:163)
	at ✽.Admin receives 500 Status with  the message and boolean success details(file:///C:/Users/User/Desktop/Git%20Project%201st%20April%20part2/RestAssured/src/test/resources/Features/I_batch_update.feature:20)
]]>
</failure>
<system-out><![CDATA[
Given Admin creates PUT Request with request body "@734384bb",3,"active","twosedet137" and "automation".passed
When Admin sends HTTPS Request  with endpoint with negative scenarios.......passed
Then Admin receives 500 Status with  the message and boolean success details.failed
]]></system-out>
</testcase>
<testcase classname="Put Request for ProgramBatch Module" name="Check if admin able to update a Batch with &quot;&lt;scenario&gt;&quot; - Example #1.6" time="0.255">
<failure type="java.lang.AssertionError" message="1 expectation failed.
Expected status code &lt;500&gt; but was &lt;404&gt;.
">
<![CDATA[java.lang.AssertionError: 1 expectation failed.
Expected status code <500> but was <404>.

	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
	at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:73)
	at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:108)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:512)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
	at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:696)
	at io.restassured.internal.ResponseSpecificationImpl.this$2$validateResponseIfRequired(ResponseSpecificationImpl.groovy)
	at jdk.internal.reflect.GeneratedMethodAccessor259.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:198)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:62)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)
	at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:135)
	at io.restassured.specification.ResponseSpecification$statusCode$0.callCurrent(Unknown Source)
	at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:143)
	at io.restassured.internal.ValidatableResponseOptionsImpl.statusCode(ValidatableResponseOptionsImpl.java:89)
	at stepDefinations.Batch_Update.admin_receives_statuscode_status_with_the_message_and_boolean_success_details(Batch_Update.java:163)
	at ✽.Admin receives 500 Status with  the message and boolean success details(file:///C:/Users/User/Desktop/Git%20Project%201st%20April%20part2/RestAssured/src/test/resources/Features/I_batch_update.feature:20)
]]>
</failure>
<system-out><![CDATA[
Given Admin creates PUT Request with request body "description",5,"active","threesedet137" and "1232@44".passed
When Admin sends HTTPS Request  with endpoint with negative scenarios.......passed
Then Admin receives 500 Status with  the message and boolean success details.failed
]]></system-out>
</testcase>
<testcase classname="Put Request for ProgramBatch Module" name="Check if admin able to update a Batch with &quot;&lt;scenario&gt;&quot; - Example #1.7" time="0.293">
<failure type="java.lang.AssertionError" message="1 expectation failed.
Expected status code &lt;500&gt; but was &lt;404&gt;.
">
<![CDATA[java.lang.AssertionError: 1 expectation failed.
Expected status code <500> but was <404>.

	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
	at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:73)
	at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:108)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:512)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
	at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:696)
	at io.restassured.internal.ResponseSpecificationImpl.this$2$validateResponseIfRequired(ResponseSpecificationImpl.groovy)
	at jdk.internal.reflect.GeneratedMethodAccessor259.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:198)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:62)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)
	at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:135)
	at io.restassured.specification.ResponseSpecification$statusCode$0.callCurrent(Unknown Source)
	at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:143)
	at io.restassured.internal.ValidatableResponseOptionsImpl.statusCode(ValidatableResponseOptionsImpl.java:89)
	at stepDefinations.Batch_Update.admin_receives_statuscode_status_with_the_message_and_boolean_success_details(Batch_Update.java:163)
	at ✽.Admin receives 500 Status with  the message and boolean success details(file:///C:/Users/User/Desktop/Git%20Project%201st%20April%20part2/RestAssured/src/test/resources/Features/I_batch_update.feature:20)
]]>
</failure>
<system-out><![CDATA[
Given Admin creates PUT Request with request body "description",5,"active","@@222@@@" and "string".passed
When Admin sends HTTPS Request  with endpoint with negative scenarios.......passed
Then Admin receives 500 Status with  the message and boolean success details.failed
]]></system-out>
</testcase>
<testcase classname="Put Request for ProgramBatch Module" name="Check if admin able to update a Batch with &quot;&lt;scenario&gt;&quot; - Example #1.8" time="0.483">
<failure type="java.lang.AssertionError" message="1 expectation failed.
Expected status code &lt;500&gt; but was &lt;404&gt;.
">
<![CDATA[java.lang.AssertionError: 1 expectation failed.
Expected status code <500> but was <404>.

	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
	at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:73)
	at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:108)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:512)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
	at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:696)
	at io.restassured.internal.ResponseSpecificationImpl.this$2$validateResponseIfRequired(ResponseSpecificationImpl.groovy)
	at jdk.internal.reflect.GeneratedMethodAccessor259.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:198)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:62)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)
	at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:135)
	at io.restassured.specification.ResponseSpecification$statusCode$0.callCurrent(Unknown Source)
	at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:143)
	at io.restassured.internal.ValidatableResponseOptionsImpl.statusCode(ValidatableResponseOptionsImpl.java:89)
	at stepDefinations.Batch_Update.admin_receives_statuscode_status_with_the_message_and_boolean_success_details(Batch_Update.java:163)
	at ✽.Admin receives 500 Status with  the message and boolean success details(file:///C:/Users/User/Desktop/Git%20Project%201st%20April%20part2/RestAssured/src/test/resources/Features/I_batch_update.feature:20)
]]>
</failure>
<system-out><![CDATA[
Given Admin creates PUT Request with request body "description",5,"@@2","threesedet137" and "string".passed
When Admin sends HTTPS Request  with endpoint with negative scenarios.......passed
Then Admin receives 500 Status with  the message and boolean success details.failed
]]></system-out>
</testcase>
<testcase classname="Put Request for ProgramBatch Module" name="Check if admin able to update a Batch with &quot;&lt;scenario&gt;&quot; - Example #1.9" time="0.262">
<failure type="java.lang.AssertionError" message="1 expectation failed.
Expected status code &lt;200&gt; but was &lt;404&gt;.
">
<![CDATA[java.lang.AssertionError: 1 expectation failed.
Expected status code <200> but was <404>.

	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
	at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:73)
	at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:108)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:512)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
	at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:696)
	at io.restassured.internal.ResponseSpecificationImpl.this$2$validateResponseIfRequired(ResponseSpecificationImpl.groovy)
	at jdk.internal.reflect.GeneratedMethodAccessor259.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:198)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:62)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)
	at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:135)
	at io.restassured.specification.ResponseSpecification$statusCode$0.callCurrent(Unknown Source)
	at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:143)
	at io.restassured.internal.ValidatableResponseOptionsImpl.statusCode(ValidatableResponseOptionsImpl.java:89)
	at stepDefinations.Batch_Update.admin_receives_statuscode_status_with_the_message_and_boolean_success_details(Batch_Update.java:163)
	at ✽.Admin receives 200 Status with  the message and boolean success details(file:///C:/Users/User/Desktop/Git%20Project%201st%20April%20part2/RestAssured/src/test/resources/Features/I_batch_update.feature:20)
]]>
</failure>
<system-out><![CDATA[
Given Admin creates PUT Request with request body "null",4,"active","threesedet137" and "string".passed
When Admin sends HTTPS Request  with endpoint with negative scenarios.......passed
Then Admin receives 200 Status with  the message and boolean success details.failed
]]></system-out>
</testcase>
<testcase classname="Put Request for ProgramBatch Module" name="Check if admin able to update a Batch with &quot;&lt;scenario&gt;&quot; - Example #1.10" time="0.028">
<failure/>
<system-out><![CDATA[
Given Admin creates PUT Request with request body "description",,"active","twosedet137" and "automation".undefined
When Admin sends HTTPS Request  with endpoint with negative scenarios.......skipped
Then Admin receives 500 Status with  the message and boolean success details.skipped
]]></system-out>
</testcase>
<testcase classname="Put Request for ProgramBatch Module" name="Check if admin able to update a Batch with &quot;&lt;scenario&gt;&quot; - Example #1.11" time="0.007">
<failure/>
<system-out><![CDATA[
Given Admin creates PUT Request with request body "description",hksjfh,"active","threesedet137" and "string".undefined
When Admin sends HTTPS Request  with endpoint with negative scenarios.......skipped
Then Admin receives 500 Status with  the message and boolean success details.skipped
]]></system-out>
</testcase>
<testcase classname="Put Request for ProgramBatch Module" name="Check if admin able to update a Batch with &quot;&lt;scenario&gt;&quot; - Example #1.1" time="0.295">
<failure type="java.lang.AssertionError" message="1 expectation failed.
Expected status code &lt;500&gt; but was &lt;404&gt;.
">
<![CDATA[java.lang.AssertionError: 1 expectation failed.
Expected status code <500> but was <404>.

	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
	at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:73)
	at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:108)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:512)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
	at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:696)
	at io.restassured.internal.ResponseSpecificationImpl.this$2$validateResponseIfRequired(ResponseSpecificationImpl.groovy)
	at jdk.internal.reflect.GeneratedMethodAccessor259.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:198)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:62)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)
	at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:135)
	at io.restassured.specification.ResponseSpecification$statusCode$0.callCurrent(Unknown Source)
	at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:143)
	at io.restassured.internal.ValidatableResponseOptionsImpl.statusCode(ValidatableResponseOptionsImpl.java:89)
	at stepDefinations.Batch_Update.admin_receives_statuscode_status_with_message(Batch_Update.java:244)
	at ✽.Admin receives 500 Status with message(file:///C:/Users/User/Desktop/Git%20Project%201st%20April%20part2/RestAssured/src/test/resources/Features/I_batch_update.feature:42)
]]>
</failure>
<system-out><![CDATA[
Given Admin creates PUT Request with  invalid batchid and request body  "description", 2,"active","onesdet137" and "automation".passed
When Admin sends HTTPS Request  with endpoints..............................passed
Then Admin receives 500 Status with message.................................failed
]]></system-out>
</testcase>
<testcase classname="Put Request for ProgramBatch Module" name="Check if admin able to update a Batch with &quot;&lt;scenario&gt;&quot; - Example #1.1" time="0.335">
<failure type="java.lang.AssertionError" message="1 expectation failed.
Expected status code &lt;400&gt; but was &lt;404&gt;.
">
<![CDATA[java.lang.AssertionError: 1 expectation failed.
Expected status code <400> but was <404>.

	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
	at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:73)
	at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:108)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:512)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
	at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:696)
	at io.restassured.internal.ResponseSpecificationImpl.this$2$validateResponseIfRequired(ResponseSpecificationImpl.groovy)
	at jdk.internal.reflect.GeneratedMethodAccessor259.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:198)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:62)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)
	at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:135)
	at io.restassured.specification.ResponseSpecification$statusCode$0.callCurrent(Unknown Source)
	at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:143)
	at io.restassured.internal.ValidatableResponseOptionsImpl.statusCode(ValidatableResponseOptionsImpl.java:89)
	at stepDefinations.Batch_Update.admin_receives_status_with_message_and_boolean_success_details(Batch_Update.java:270)
	at ✽.Admin receives 400 Status with message and boolean success details(file:///C:/Users/User/Desktop/Git%20Project%201st%20April%20part2/RestAssured/src/test/resources/Features/I_batch_update.feature:53)
]]>
</failure>
<system-out><![CDATA[
Given Admin creates PUT Request with request body and invalid endpoint "Batch" and 1.passed
When Admin sends HTTPS Request  with invalid endpoints......................passed
Then Admin receives 400 Status with message and boolean success details.....failed
]]></system-out>
</testcase>
<testcase classname="Put Request for ProgramBatch Module" name="Check if admin able to update  Batch with valid batchID and mandatory fields in request body without authorization - Example #1.1" time="0.25">
<system-out><![CDATA[
Given Admin creates PUT Request with  the valid BatchId and Data "Batch" and 1.passed
When Admin sends HTTPS Request  with endpoint and invalid Auth..............passed
Then Admin receives  the 401 and "Unauthorized" unauthorized................passed
]]></system-out>
</testcase>
<testcase classname="Get Request for ProgramBatch Module" name="Check if admin able to retrive a batch after the programID is deleted - Example #1.1" time="0.295">
<system-out><![CDATA[
Given Admin creates GET Request with  deleted program id....................passed
When Admin sends HTTPS Request with endpoint and deleted programId..........passed
Then Admin receives 404  Not Found Status with message and boolean success details.passed
]]></system-out>
</testcase>
<testcase classname="User Module" name="Check if user is able to create a new Admin,Staff &amp; Student with valid endpoint and request body with mandatory fields - Example #1.1" time="0.396">
<system-out><![CDATA[
Given User creates POST Request for the LMS with request body "USER" and 1..passed
When User sends "LMScreateUser" with "Post" Request with endpoint...........passed
Then User receives 201 Created Status with response body....................passed
]]></system-out>
</testcase>
<testcase classname="User Module" name="Check if user is able to create a new Admin,Staff &amp; Student with valid endpoint and request body with mandatory fields - Example #1.2" time="0.271">
<system-out><![CDATA[
Given User creates POST Request for the LMS with request body "USER" and 2..passed
When User sends "LMScreateUser" with "Post" Request with endpoint...........passed
Then User receives 201 Created Status with response body....................passed
]]></system-out>
</testcase>
<testcase classname="User Module" name="Check if user is able to create a new Admin,Staff &amp; Student with valid endpoint and request body with mandatory fields - Example #1.3" time="0.396">
<system-out><![CDATA[
Given User creates POST Request for the LMS with request body "USER" and 3..passed
When User sends "LMScreateUser" with "Post" Request with endpoint...........passed
Then User receives 201 Created Status with response body....................passed
]]></system-out>
</testcase>
<testcase classname="User Module" name="Check if user is able to get the user" time="0.521">
<failure type="java.lang.RuntimeException" message="com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field &quot;timestamp&quot; (class POJO.UserGetPojo), not marked as ignorable (2 known properties: &quot;user&quot;, &quot;roles&quot;])
 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 15] (through reference chain: POJO.UserGetPojo[&quot;timestamp&quot;])">
<![CDATA[java.lang.RuntimeException: com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "timestamp" (class POJO.UserGetPojo), not marked as ignorable (2 known properties: "user", "roles"])
 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 15] (through reference chain: POJO.UserGetPojo["timestamp"])
	at io.restassured.internal.path.json.mapping.JsonPathJackson2ObjectDeserializer.deserialize(JsonPathJackson2ObjectDeserializer.java:30)
	at io.restassured.path.json.mapping.JsonPathObjectDeserializer$deserialize.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:45)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:139)
	at io.restassured.internal.mapping.Jackson2Mapper.deserialize(Jackson2Mapper.groovy:58)
	at io.restassured.mapper.ObjectMapper$deserialize$0.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:45)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:139)
	at io.restassured.internal.mapping.ObjectMapping.parseWithJackson2(ObjectMapping.groovy:254)
	at io.restassured.internal.mapping.ObjectMapping$parseWithJackson2$1.callStatic(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallStatic(CallSiteArray.java:53)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callStatic(AbstractCallSite.java:217)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callStatic(AbstractCallSite.java:240)
	at io.restassured.internal.mapping.ObjectMapping.deserialize(ObjectMapping.groovy:56)
	at io.restassured.internal.mapping.ObjectMapping$deserialize$0.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:45)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)
	at io.restassured.internal.RestAssuredResponseOptionsGroovyImpl.as(RestAssuredResponseOptionsGroovyImpl.groovy:229)
	at io.restassured.internal.RestAssuredResponseOptionsImpl.as(RestAssuredResponseOptionsImpl.java:169)
	at stepDefinations.User.user_validate_the_response_json(User.java:241)
	at ✽.User validate the response json(file:///C:/Users/User/Desktop/Git%20Project%201st%20April%20part2/RestAssured/src/test/resources/Features/K_user.feature:77)
Caused by: com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "timestamp" (class POJO.UserGetPojo), not marked as ignorable (2 known properties: "user", "roles"])
 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 15] (through reference chain: POJO.UserGetPojo["timestamp"])
	at com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException.from(UnrecognizedPropertyException.java:61)
	at com.fasterxml.jackson.databind.DeserializationContext.handleUnknownProperty(DeserializationContext.java:1153)
	at com.fasterxml.jackson.databind.deser.std.StdDeserializer.handleUnknownProperty(StdDeserializer.java:2241)
	at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownProperty(BeanDeserializerBase.java:1793)
	at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownVanilla(BeanDeserializerBase.java:1771)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:316)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:177)
	at com.fasterxml.jackson.databind.deser.DefaultDeserializationContext.readRootValue(DefaultDeserializationContext.java:342)
	at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:4905)
	at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3848)
	at io.restassured.internal.path.json.mapping.JsonPathJackson2ObjectDeserializer.deserialize(JsonPathJackson2ObjectDeserializer.java:28)
	at io.restassured.path.json.mapping.JsonPathObjectDeserializer$deserialize.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:45)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:139)
	at io.restassured.internal.mapping.Jackson2Mapper.deserialize(Jackson2Mapper.groovy:58)
	at io.restassured.mapper.ObjectMapper$deserialize$0.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:45)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:139)
	at io.restassured.internal.mapping.ObjectMapping.parseWithJackson2(ObjectMapping.groovy:254)
	at io.restassured.internal.mapping.ObjectMapping$parseWithJackson2$1.callStatic(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallStatic(CallSiteArray.java:53)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callStatic(AbstractCallSite.java:217)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callStatic(AbstractCallSite.java:240)
	at io.restassured.internal.mapping.ObjectMapping.deserialize(ObjectMapping.groovy:56)
	at io.restassured.internal.mapping.ObjectMapping$deserialize$0.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:45)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)
	at io.restassured.internal.RestAssuredResponseOptionsGroovyImpl.as(RestAssuredResponseOptionsGroovyImpl.groovy:229)
	at io.restassured.internal.RestAssuredResponseOptionsImpl.as(RestAssuredResponseOptionsImpl.java:169)
	at stepDefinations.User.user_validate_the_response_json(User.java:241)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at io.cucumber.java.Invoker.doInvoke(Invoker.java:66)
	at io.cucumber.java.Invoker.invoke(Invoker.java:24)
	at io.cucumber.java.AbstractGlueDefinition.invokeMethod(AbstractGlueDefinition.java:47)
	at io.cucumber.java.JavaStepDefinition.execute(JavaStepDefinition.java:29)
	at io.cucumber.core.runner.CoreStepDefinition.execute(CoreStepDefinition.java:66)
	at io.cucumber.core.runner.PickleStepDefinitionMatch.runStep(PickleStepDefinitionMatch.java:63)
	at io.cucumber.core.runner.ExecutionMode$1.execute(ExecutionMode.java:10)
	at io.cucumber.core.runner.TestStep.executeStep(TestStep.java:84)
	at io.cucumber.core.runner.TestStep.run(TestStep.java:56)
	at io.cucumber.core.runner.PickleStepTestStep.run(PickleStepTestStep.java:51)
	at io.cucumber.core.runner.TestCase.run(TestCase.java:84)
	at io.cucumber.core.runner.Runner.runPickle(Runner.java:75)
	at io.cucumber.junit.PickleRunners$NoStepDescriptions.lambda$run$0(PickleRunners.java:151)
	at io.cucumber.core.runtime.CucumberExecutionContext.lambda$runTestCase$5(CucumberExecutionContext.java:137)
	at io.cucumber.core.runtime.RethrowingThrowableCollector.executeAndThrow(RethrowingThrowableCollector.java:23)
	at io.cucumber.core.runtime.CucumberExecutionContext.runTestCase(CucumberExecutionContext.java:137)
	at io.cucumber.junit.PickleRunners$NoStepDescriptions.run(PickleRunners.java:148)
	at io.cucumber.junit.FeatureRunner.runChild(FeatureRunner.java:144)
	at io.cucumber.junit.FeatureRunner.runChild(FeatureRunner.java:28)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at io.cucumber.junit.FeatureRunner.run(FeatureRunner.java:137)
	at io.cucumber.junit.Cucumber.runChild(Cucumber.java:196)
	at io.cucumber.junit.Cucumber.runChild(Cucumber.java:89)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at io.cucumber.core.runtime.CucumberExecutionContext.lambda$runFeatures$6(CucumberExecutionContext.java:148)
	at io.cucumber.core.runtime.CucumberExecutionContext.execute(CucumberExecutionContext.java:163)
	at io.cucumber.core.runtime.CucumberExecutionContext.runFeatures(CucumberExecutionContext.java:146)
	at io.cucumber.junit.Cucumber$StartAndFinishTestRun.evaluate(Cucumber.java:226)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:93)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:40)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:529)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:756)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:452)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:210)
]]>
</failure>
<system-out><![CDATA[
Given User creates get request with Authorization...........................passed
When User sends "LMSgetSingleUser" with "Get" Request with endpoint.........passed
Then User receives 200 Created Status with response body....................passed
And User validate the response json.........................................failed
]]></system-out>
</testcase>
<testcase classname="User Module" name="Check if admin is able to update UserRoleProgramBatchStatus" time="0.242">
<system-out><![CDATA[
Given User creates PUT request with all fields..............................passed
When User sends "LMSupdateUserRoleProgramBatchStatus" with "Put" Request with endpoint.passed
Then User receives 200 Created Status with response body....................passed
]]></system-out>
</testcase>
<testcase classname="User Module" name="Check if Admin able to delete a Staff with valid Admin Id" time="0.686">
<failure type="java.lang.NullPointerException" message="Cannot invoke &quot;io.restassured.specification.RequestSpecification.when()&quot; because &quot;stepDefinations.User.res7&quot; is null">
<![CDATA[java.lang.NullPointerException: Cannot invoke "io.restassured.specification.RequestSpecification.when()" because "stepDefinations.User.res7" is null
	at stepDefinations.User.user_sends_with_request_with_endpoint(User.java:191)
	at ✽.User sends "LMSdeleteUser" with "Delete" Request with endpoint(file:///C:/Users/User/Desktop/Git%20Project%201st%20April%20part2/RestAssured/src/test/resources/Features/K_user.feature:169)
]]>
</failure>
<system-out><![CDATA[
Given User creates post Request to delete Staff details.....................passed
When User sends "LMSdeleteUser" with "Delete" Request with endpoint.........failed
Then User receives 200 Created Status with response body....................skipped
]]></system-out>
</testcase>
<testcase classname="User Module" name="Check if Admin able to delete a Student with valid Admin Id" time="0.458">
<system-out><![CDATA[
Given User creates post Request to delete Student details...................passed
When User sends "LMSdeleteUser" with "Delete" Request with endpoint.........passed
Then User receives 200 Created Status with response body....................passed
]]></system-out>
</testcase>
<testcase classname="UserRoleProgramBatchMap Module" name="Check if admin is able to retreive all admin with assigned program batches" time="0.483">
<failure type="java.lang.NullPointerException" message="Cannot invoke &quot;Object.toString()&quot; because the return value of &quot;io.restassured.path.json.JsonPath.get(String)&quot; is null">
<![CDATA[java.lang.NullPointerException: Cannot invoke "Object.toString()" because the return value of "io.restassured.path.json.JsonPath.get(String)" is null
	at Utilities.ReusableMethod.getJsonPath(ReusableMethod.java:70)
	at stepDefinations.UseRoleProgramBatchMap.user_role_program_batch_map_receives_created_status_with_response_body(UseRoleProgramBatchMap.java:73)
	at ✽.UserRoleProgramBatchMap receives 200 Created Status with response body(file:///C:/Users/User/Desktop/Git%20Project%201st%20April%20part2/RestAssured/src/test/resources/Features/L_UserRoleProgramBatchMap.feature:12)
]]>
</failure>
<system-out><![CDATA[
Given UserRoleProgramBatchMap creates get request with Authorization........passed
When UserRoleProgramBatchMap sends "LMSgetProgramBatche" with "Get" Request with endpoint.passed
Then UserRoleProgramBatchMap receives 200 Created Status with response body.failed
]]></system-out>
</testcase>
<testcase classname="UserRoleProgramBatchMap Module" name="Check if admin is able to retreive assigned program batches for valid AdminId" time="0.429">
<failure type="java.lang.NullPointerException" message="Cannot invoke &quot;Object.toString()&quot; because the return value of &quot;io.restassured.path.json.JsonPath.get(String)&quot; is null">
<![CDATA[java.lang.NullPointerException: Cannot invoke "Object.toString()" because the return value of "io.restassured.path.json.JsonPath.get(String)" is null
	at Utilities.ReusableMethod.getJsonPath(ReusableMethod.java:70)
	at stepDefinations.UseRoleProgramBatchMap.user_role_program_batch_map_receives_created_status_with_response_body(UseRoleProgramBatchMap.java:73)
	at ✽.UserRoleProgramBatchMap receives 200 Created Status with response body(file:///C:/Users/User/Desktop/Git%20Project%201st%20April%20part2/RestAssured/src/test/resources/Features/L_UserRoleProgramBatchMap.feature:17)
]]>
</failure>
<system-out><![CDATA[
Given UserRoleProgramBatchMap creates get request with Authorization........passed
When UserRoleProgramBatchMap sends "LMSgetUserProgramBatch" with "Get" Request with endpoint.passed
Then UserRoleProgramBatchMap receives 200 Created Status with response body.failed
]]></system-out>
</testcase>
<testcase classname="UserRoleProgramBatchMap Module" name="Check if admin is able to retreive assigned program batches for Invalid AdminId" time="0.604">
<failure type="java.lang.NullPointerException" message="Cannot invoke &quot;Object.toString()&quot; because the return value of &quot;io.restassured.path.json.JsonPath.get(String)&quot; is null">
<![CDATA[java.lang.NullPointerException: Cannot invoke "Object.toString()" because the return value of "io.restassured.path.json.JsonPath.get(String)" is null
	at Utilities.ReusableMethod.getJsonPath(ReusableMethod.java:70)
	at stepDefinations.UseRoleProgramBatchMap.user_role_program_batch_map_receives_created_status_with_response_body(UseRoleProgramBatchMap.java:73)
	at ✽.UserRoleProgramBatchMap receives 404 Created Status with response body(file:///C:/Users/User/Desktop/Git%20Project%201st%20April%20part2/RestAssured/src/test/resources/Features/L_UserRoleProgramBatchMap.feature:23)
]]>
</failure>
<system-out><![CDATA[
Given UserRoleProgramBatchMap creates get request with Authorization........passed
When UserRoleProgramBatchMap sends "LMSgetUserProgramBatchInvalid" with "Get" Request with endpoint.passed
Then UserRoleProgramBatchMap receives 404 Created Status with response body.failed
]]></system-out>
</testcase>
<testcase classname="UserRoleProgramBatchMap Module" name="Check if admin is able to delete the program batch for a Admin" time="0.072">
<failure type="java.lang.NullPointerException" message="Cannot invoke &quot;io.restassured.specification.RequestSpecification.when()&quot; because &quot;this.resUserRole&quot; is null">
<![CDATA[java.lang.NullPointerException: Cannot invoke "io.restassured.specification.RequestSpecification.when()" because "this.resUserRole" is null
	at stepDefinations.UseRoleProgramBatchMap.user_role_program_batch_map_sends_with_request_with_endpoint(UseRoleProgramBatchMap.java:55)
	at ✽.UserRoleProgramBatchMap sends "LMSDeleteProgramBatche" with "Delete" Request with endpoint(file:///C:/Users/User/Desktop/Git%20Project%201st%20April%20part2/RestAssured/src/test/resources/Features/L_UserRoleProgramBatchMap.feature:29)
]]>
</failure>
<system-out><![CDATA[
Given UserRoleProgramBatchMap creates delete request with Authorization.....passed
When UserRoleProgramBatchMap sends "LMSDeleteProgramBatche" with "Delete" Request with endpoint.failed
Then UserRoleProgramBatchMap receives 200 Created Status with response body.skipped
]]></system-out>
</testcase>
<testcase classname="UserRoleProgramBatchMap Module" name="Check if UserRoleProgramBatchMap is able to delete the program batch for invalid Admin" time="0.035">
<failure type="java.lang.NullPointerException" message="Cannot invoke &quot;io.restassured.specification.RequestSpecification.when()&quot; because &quot;this.resUserRole&quot; is null">
<![CDATA[java.lang.NullPointerException: Cannot invoke "io.restassured.specification.RequestSpecification.when()" because "this.resUserRole" is null
	at stepDefinations.UseRoleProgramBatchMap.user_role_program_batch_map_sends_with_request_with_endpoint(UseRoleProgramBatchMap.java:59)
	at ✽.UserRoleProgramBatchMap sends "LMSDeleteProgramBatcheInvalid" with "Delete" Request with endpoint(file:///C:/Users/User/Desktop/Git%20Project%201st%20April%20part2/RestAssured/src/test/resources/Features/L_UserRoleProgramBatchMap.feature:36)
]]>
</failure>
<system-out><![CDATA[
Given UserRoleProgramBatchMap creates delete request with Authorization.....passed
When UserRoleProgramBatchMap sends "LMSDeleteProgramBatcheInvalid" with "Delete" Request with endpoint.failed
Then UserRoleProgramBatchMap receives 404 Created Status with response body.skipped
]]></system-out>
</testcase>
<testcase classname="Get Request for ProgramBatch Module" name="Check if admin able to delete a Batch with valid Batch ID - Example #1.1" time="0.421">
<failure type="java.lang.AssertionError" message="1 expectation failed.
Expected status code &lt;200&gt; but was &lt;404&gt;.
">
<![CDATA[java.lang.AssertionError: 1 expectation failed.
Expected status code <200> but was <404>.

	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
	at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:73)
	at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:108)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:512)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
	at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:696)
	at io.restassured.internal.ResponseSpecificationImpl.this$2$validateResponseIfRequired(ResponseSpecificationImpl.groovy)
	at jdk.internal.reflect.GeneratedMethodAccessor259.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:198)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:62)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)
	at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:135)
	at io.restassured.specification.ResponseSpecification$statusCode$0.callCurrent(Unknown Source)
	at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:143)
	at io.restassured.internal.ValidatableResponseOptionsImpl.statusCode(ValidatableResponseOptionsImpl.java:89)
	at stepDefinations.Delete_BatchId.admin_receives_ok_status_with_message(Delete_BatchId.java:40)
	at ✽.Admin receives 200 Ok status with message(file:///C:/Users/User/Desktop/Git%20Project%201st%20April%20part2/RestAssured/src/test/resources/Features/M_DeleteBatchId.feature:9)
]]>
</failure>
<system-out><![CDATA[
Given Admin creates DELETE Request with valid BatchId.......................passed
When Admin sends HTTPS Request  with endpoint and batchId...................passed
Then Admin receives 200 Ok status with message..............................failed
]]></system-out>
</testcase>
<testcase classname="Get Request for ProgramBatch Module" name="Check if admin able to delete a Batch with invalid endpoint - Example #1.1" time="0.191">
<failure type="java.lang.AssertionError" message="1 expectation failed.
Expected status code &lt;400&gt; but was &lt;404&gt;.
">
<![CDATA[java.lang.AssertionError: 1 expectation failed.
Expected status code <400> but was <404>.

	at java.base/jdk.internal.reflect.GeneratedConstructorAccessor119.newInstance(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
	at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:73)
	at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:108)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:512)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
	at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:696)
	at io.restassured.internal.ResponseSpecificationImpl.this$2$validateResponseIfRequired(ResponseSpecificationImpl.groovy)
	at jdk.internal.reflect.GeneratedMethodAccessor259.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:198)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:62)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)
	at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:135)
	at io.restassured.specification.ResponseSpecification$statusCode$0.callCurrent(Unknown Source)
	at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:143)
	at io.restassured.internal.ValidatableResponseOptionsImpl.statusCode(ValidatableResponseOptionsImpl.java:89)
	at stepDefinations.Delete_BatchId.admin_receives_not_found(Delete_BatchId.java:67)
	at ✽.Admin receives 400 not found(file:///C:/Users/User/Desktop/Git%20Project%201st%20April%20part2/RestAssured/src/test/resources/Features/M_DeleteBatchId.feature:19)
]]>
</failure>
<system-out><![CDATA[
Given Admin creates DELETE Request with valid BatchId and invalid enpoint...passed
When Admin sends HTTPS Request  with invalid endpoint and batchId...........passed
Then Admin receives 400 not found...........................................failed
]]></system-out>
</testcase>
<testcase classname="Get Request for ProgramBatch Module" name="Check if admin able to delete a Batch with invalid Batch ID - Example #1.1" time="0.261">
<system-out><![CDATA[
Given Admin creates DELETE Request with invalid BatchId.....................passed
When Admin sends HTTPS Request  with endpoint and invalid batchId...........passed
Then Admin receives 404 Not Found Status with message and boolean success details.passed
]]></system-out>
</testcase>
<testcase classname="Get Request for ProgramBatch Module" name="Check if admin able to delete a Batch without authorization - Example #1.1" time="0.192">
<system-out><![CDATA[
Given Admin creates DELETE Request with valid BatchId and unAuth............passed
When Admin sends HTTPS Request  with endpoint and without authorization.....passed
Then Admin receives 401 and message  "Unauthorized".........................passed
]]></system-out>
</testcase>
</testsuite>
